aws/table_aws_ecs_task_definition.go

```diff
@@ -0,0 +1,315 @@
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.PlacementConstraints"),
+			},
+			{
+				Name:        "proxy_configuration",
+				Description: "The configuration details for the App Mesh proxy.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.ProxyConfiguration"),
+			},
+			{
+				Name:        "requires_attributes",
+				Description: "The container instance attributes required by your task.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.RequiresAttributes"),
+			},
+			{
+				Name:        "requires_compatibilities",
+				Description: "The launch type the task requires. If no value is specified, it will default to EC2. Valid values include EC2 and FARGATE.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.RequiresCompatibilities"),
+			},
+			{
+				Name:        "volumes",
+				Description: "The list of volume definitions for the task.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.Volumes"),
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags associated with target group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromP(getAwsEcsTaskDefinitionTurbotData, "Title"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromP(getAwsEcsTaskDefinitionTurbotData, "Tags"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsTaskDefinition,
+				Transform:   transform.FromField("TaskDefinition.TaskDefinitionArn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listEcsTaskDefinitions(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	// Create Session
+	svc, err := EcsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+	params := &ecs.ListTaskDefinitionsInput{}
+	pagesLeft := true
+
+	for pagesLeft {
+		result, err := svc.ListTaskDefinitions(params)
```

Comment:
Can't we use `ListTaskDefinitionsPages` for pagination?
https://github.com/aws/aws-sdk-go/blob/67f532fc99ebc262a50fa05500b3462065b75569/service/ecs/api.go#L3317